README.md

```diff
@@ -72,6 +74,10 @@ Where:
 Enable open loop mode. When enabled, the benchmark client will not
 provide backpressure when resource limits are hit.
 
+--h1-connection-reuse-strategy <HOT|FAIR>
+Choose picking the most recently used, or least-recently-used
+connections for re-use.(default: hot).
```

Comment:
Do you think this is the best "out of box" experience? Would fair have some advantage in making it harder to do the wrong thing?

Reply:
Initially I wanted to propose `FAIR` as a new default, because in certain scenario's it may help the distribute load better across available computing capacity (threads/processes/machines).
But on the other hand, I guess `HOT` is the more realistic approach, as most client side (pool) implementations follow that pattern I guess.. 
So I was kind of torn and backed out the update to the default behavior.  (I also mulled over adding `RANDOM` here as an more challenging pattern, which may be representative for certain use cases as well.. so I ended up thinking that what is best here this kind of might just depend on the kind of traffic that you want to simulate).
Given all that, do you have any thoughts here?

Reply:
I think "fair" is a better out-of-the-box experience for characterizing performance; when a user wants to get to the nuances of connection reuse and modeling the behavior of other clients, we can advise them to switch to "hot".